Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.yml

```diff
@@ -181,6 +181,12 @@ script:
       name: offset
       required: false
```

Comment:
lets remember to change name and id before sending to them

Reply:
ofc.